From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 7 Jul 2011 10:49:54 -0700
Subject: staging: fix usbip printk format warning

commit 6394c5a0379d0733396edd9452e31282e0684a3c upstream.

Fix usbip printk format warning for size_t:

drivers/staging/usbip/stub_tx.c:236: warning: format '%lu' expects type 'long unsigned int', but argument 4 has type 'size_t'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
[bwh: Backported to 2.6.32: adjusted context]
---
--- a/drivers/staging/usbip/stub_tx.c
+++ b/drivers/staging/usbip/stub_tx.c
@@ -237,7 +237,7 @@ static int stub_send_ret_submit(struct stub_device *sdev)
 
 			if (txsize != sizeof(pdu_header) + urb->actual_length) {
 				dev_err(&sdev->interface->dev,
-					"actual length of urb (%d) does not match iso packet sizes (%d)\n",
+					"actual length of urb (%d) does not match iso packet sizes %zu\n",
 					urb->actual_length, txsize-sizeof(pdu_header));
 				kfree(iov);
 				usbip_event_add(&sdev->ud, SDEV_EVENT_ERROR_TCP);
